joomla/joomla-cms
# Push footer to bottom and fix content vertically centered and fix scroll to top

Pull Request for Issue #35003 and replacement of PR #35004

### Summary of Changes
Added holy grail layout to the Cassiopeia template. That way the footer can be pushed to the bottom if there is less content then the screen height. Changed the way the scroll to top is implemented. In the process also removed some obsolete CSS code.

### Testing Instructions
See issue #35003 and description of PR #34849
Please test with and without modules and content and in different display sizes. Also test with short content and some module in footer and header positions.

Don't forget to test the error page as well. This can be done bij typing a URL which doesn't exist on the frontend of the site. For instance https://[test site url]/index.php/thislinkdoesnotexist. This will generate a 404 error, page not found. Make sure the footer is also be on the bottom of the site.

### Actual result BEFORE applying this Pull Request

### Expected result AFTER applying this Pull Request

### Documentation Changes Required
